src/vm/eetwain.cpp

```diff
@@ -5169,7 +5169,11 @@ OBJECTREF EECodeManager::GetInstance( PREGDISPLAY    pContext,
     if (info.ebpFrame)
     {
         _ASSERTE(stackDepth == 0);
+#if defined(_TARGET_X86_) && defined(FEATURE_PAL)
+        taArgBase = GetCallerSp(pContext) - 2 * sizeof(TADDR);
+#else
         taArgBase = *pContext->GetEbpLocation();
```

Comment:
@seanshpark It was ``*pContext->pEbp;``, but revised as the current form while x86/Linux bring up (#9121).

For 5180, it takes care of ``esp``-based frames (not ``ebp``-based frames), and thus it would be better to leave it as the current form.